com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -627,23 +671,36 @@ internal int[] rendererIndexList
             }
         }
 
+        /// <summary>
+        /// Specifies if the Camera Depth Texture should be used for this <c>UniversalRenderPipelineAsset</c>.
+        /// </summary>
         public bool supportsCameraDepthTexture
         {
             get { return m_RequireDepthTexture; }
             set { m_RequireDepthTexture = value; }
         }
 
+        /// <summary>
+        /// Specifies if the Camera Opaque Texture should be used for this <c>UniversalRenderPipelineAsset</c>.
+        /// </summary>
         public bool supportsCameraOpaqueTexture
         {
             get { return m_RequireOpaqueTexture; }
             set { m_RequireOpaqueTexture = value; }
         }
 
+        /// <summary>
+        /// Returns the downsampling used for this <c>UniversalRenderPipelineAsset</c>.
+        /// </summary>
         public Downsampling opaqueDownsampling
         {
             get { return m_OpaqueDownsampling; }
         }
 
+        /// <summary>
+        /// This is to check if this <c>UniversalRenderPipelineAsset</c> supports terrain holes.
```

Comment:
Suggestion, replace "This is to check if", with "When enabled, ... " , "This settings controls ... ", "Specifies if... " or something similar.

Reply:
Updated it to:
This settings controls if the asset <c>UniversalRenderPipelineAsset</c> supports terrain holes.